nntrainer/src/neuralnet.cpp

```diff
@@ -734,7 +740,7 @@ int NeuralNetwork::train() {
 
 int NeuralNetwork::checkValidation() {
   int status = ML_ERROR_NONE;
-  if (config.empty())
+  if (!config.empty())
```

Comment:
`checkValidation` would be skipped if config is not empty without checking layers.
If `config.empty()` is a valid status, I think this line can be deleted.
```suggestion
  if (config.empty())
     return ML_ERROR_INVALID_PARAMETER;
```

Reply:
Well, actually, there are two options to run the neural network, run with configuration file and without configuration. If there is configuration file, then there is no need to check the layer validation. just return no error. if there is no configuration file, then check the validation of layer, and if there is error, then return error. if not return no error.

Reply:
Thanks for the information!